### PR TITLE
Fill plan properties in Http Headers and ServiceBus Message Properties instead of body and increased the major version.

### PR DESCRIPTION
**Pull request by sureshtadisetty**
Fill plan properties in Http Headers and ServiceBus Message Properties instead of body and increased the major version.

### CODE REVIEW

**Comment by vithati** [comment]
Do changes in azure function also.

File: Tasks/InvokeRestApi/task.json [hunk 1]
```diff
@@ -58,12 +58,12 @@
       "name":"headers",
       "type":"multiLine",
       "label":"Headers",
-      "required":"false",
-      "defaultValue":"{\n\"Content-Type\":\"application/json\"\n}",
+      "required":"true",
```
Comment (vithati) [inline Tasks/InvokeRestApi/task.json]: why it is mandatory?

File: Tasks/InvokeRestApi/task.loc.json [hunk 2]
```diff
@@ -61,12 +61,12 @@
       "name": "headers",
       "type": "multiLine",
       "label": "ms-resource:loc.input.label.headers",
-      "required": "false",
-      "defaultValue": "{\n\"Content-Type\":\"application/json\"\n}",
+      "required": "true",
+      "defaultValue": "{\n\"Content-Type\":\"application/json\", \n\"VstsUrl\": \"$(system.CollectionUri)\", \n\"ProjectId\": \"$(system.teamProjectId)\", \n\"HubName\": \"$(system.HubName)\", \n\"PlanId\": \"$(system.planId)\", \n\"JobId\": \"$(system.jobId)\", \n\"TimelineId\": \"$(system.timelineId)\", \n\"TaskInstanceId\": \"$(system.TaskInstanceId)\", \n\"AuthToken\": \"$(system.AccessToken)\"\n}",
```
Comment (vithati) [inline Tasks/InvokeRestApi/task.loc.json]: i think system.HubName not exist. HubName is SYSTEM.HOSTTYPE.

File: Tasks/PublishToAzureServiceBus/Strings/resources.resjson/en-US/resources.resjson [hunk 3]
```diff
@@ -7,6 +7,8 @@
   "loc.input.help.connectedServiceName": "Select a Azure Service Bus connection.",
   "loc.input.label.messageBody": "Message body",
   "loc.input.help.messageBody": "Enter the json messageBody.",
+  "loc.input.label.messageProperties": "Message properties",
+  "loc.input.help.messageProperties": "Enter the json messageProperties.",
```
Comment (vithati) [inline Tasks/PublishToAzureServiceBus/Strings/resources.resjson/en-US/resources.resjson]: Enter the json Message Properties.

File: Tasks/InvokeRestApi/task.json (hunk 1 above)
Comment (sureshtadisetty) [reply Tasks/InvokeRestApi/task.json]: We're filling task* related data in headers, can be used to update task timeline record from customer's external service.

File: Tasks/PublishToAzureServiceBus/Strings/resources.resjson/en-US/resources.resjson (hunk 3 above)
Comment (sureshtadisetty) [reply Tasks/PublishToAzureServiceBus/Strings/resources.resjson/en-US/resources.resjson]: fixed.

**Pull request closed without merging**